tests/test_rates.py

```diff
@@ -19,15 +31,49 @@
 )
 
 
-def test_token_rates():
+token_rates_testdata = [
```

Comment:
Let's make the constant uppercase to conform with https://peps.python.org/pep-0008/#constants

Reply:
fixed